services/movie_session.py

```diff
@@ -0,0 +1,54 @@
+from datetime import datetime
+
+from django.db.models import QuerySet
+
+from db.models import MovieSession
+
+
+def create_movie_session(movie_show_time: datetime,
+                         movie_id: int,
+                         cinema_hall_id: int) -> MovieSession:
+    new_movie_session = MovieSession(
+        show_time=movie_show_time,
+        movie_id=movie_id,
+        cinema_hall_id=cinema_hall_id
+    )
+
+    new_movie_session.save()
+
+    return new_movie_session
+
+
+def get_movies_sessions(session_date: str = None) -> QuerySet:
+    if not session_date:
+        return MovieSession.objects.all()
+
+    desired_date = datetime.strptime(session_date, "%Y-%m-%d").date()
+
+    return MovieSession.objects.filter(show_time__date=desired_date)

```

Comment:
`desired_date` is unnecessary
you can just filter with `show_time__date=session_date`